rfcs/0007-scoring-system-and-network-security/0007-scoring-system-and-network-security.md

```diff
@@ -46,8 +46,8 @@ CKB client should implement the scoring system and following security strategies
 * `Peer` - Other nodes connected through the network
 * `PeerInfo` - A data struct used for describing information of `Peer`
 * `PeerStore` - A component used to store `PeerInfo`
-* `outbound peer` - describe a peer which we initiate a connection.
-* `inbound peer` - describe a peer which the peer initiates a connection.
+* `outbound peer` - describe a peer which initiatively generates a connection.
```

Comment:
Typically we do not say `generate a connection`, `initiate a connection` is more generally used.